### PR TITLE
Customer bug 1860094 automation

### PR DESCRIPTION
**Pull request by SrinivasaBharath**
# Description

Please include Automation development guidelines. Source of Test case - New Feature/Regression Test/Close loop of customer BZs

# Checklist:

- [x] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [ ] Review the automation design
- [x] Implement the test script and perform test runs
- [x] Submit PR for code review and approve
- [x] Update Polarin Test with Automation script details and update automation fields
- [x] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test

### CODE REVIEW

**Comment by SrinivasaBharath** [comment]
1860094 bug is extension of the the bug 1829646.Please let me know do I need to change the file name which I need to include the 1860094 bug .

Authoring test case in polarion is in progress.

**Comment by SrinivasaBharath** [comment]
Created Polarion Test case.
https://polarion.engineering.redhat.com/polarion/redirect/project/CEPH/workitem?id=CEPH-83573589

**Comment by rakeshgm** [comment]
@SrinivasaBharath can you fix the travis-ci errors.

**Comment by SrinivasaBharath** [comment]
travis-ci errors are fixed.

File: tests/rados/test_bz_1829646.py [hunk 1]
```diff
@@ -128,8 +128,21 @@ def run(ceph_cluster, **kw):
...
+    #   ceph commands or not
+
     if max_size_increase > final_size_change:
         log.info(f"The monDB map was trimmed by : {abs(max_size_increase - final_size_change)}")
+        ceph_health_status = get_status_from_ceph_report(mon_node=controller, operation='health')
+        # Checking that command executed or not.If executed then dictionary output should not be empty.
+        if(bool(ceph_health_status)):
+            log.info("The ceph command executed after after succesful execution of monDB trim")
+        else:
+            log.error("The ceph command not executed after after succesful execution of monDB trim")
```
Comment (rakeshgm) [inline tests/rados/test_bz_1829646.py:145]: I think its best to check the return value of `ceph_health_status` and then do a log.error. bool(ceph_heath_status) only gives if value is present in it or None, we need to check the value and then proceed.

**Comment by rakeshgm** [comment]
@SrinivasaBharath any update here.

**Comment by rakeshgm** [comment]
@SrinivasaBharath, and update ?

**Comment by SrinivasaBharath** [comment]
> @SrinivasaBharath, and update ?

Working on this. I will provide the  solution.

**Comment by SrinivasaBharath** [comment]
Closing this PR as this is not valid due to cephadm.

**Pull request closed without merging**